packages/common/test/core/getTypesFromEntityType.ts

```diff
@@ -0,0 +1,34 @@
+import { HubEntity, HubEntityType } from "../../src/core/types";
+import { getTypesFromEntityType } from "../../src/core/getTypesFromEntityType";
+
+describe("getTypesFromEntityType:", () => {
+  it("returns the item type(s) for all Hub entity types", () => {
+    const types = [
```

Comment:
Random -- but should this file be named `getTypesFromEntityType.test.ts`?

Reply:
good catch!